cypress/support/e2e.js

```diff
@@ -51,9 +51,12 @@ Cypress.Commands.add('addContext', (context) => {
 Cypress.Commands.overwrite('log', (orig, string, options) => {
   orig(string, options);
   const dateString = new Date().toLocaleString().replace(',', '');
-  cy.addContext('[' + dateString + '][' + options + ']:[' + JSON.stringify(string) + ']');
+
+  typeof string == 'object' ? (string = JSON.stringify(string)) : string;
```

Comment:
Minor semantic naming confusion here, `string == object`? That is 2 different datatypes. Could use a more meaningful name

Reply:
Yea "string" was the existing variable which we just removed the stringify operator off of and conditionally stringified only in case of it being an object type...i would say maybe we can name it something like "message" and just change all of the references inside the command overwrite to match.